### PR TITLE
Bug 2219136: Values on graph go out of tile

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
image

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar, @GowthamShanmugam Please review

**Comment by SanjalKatiyar** [comment]
fix build test...

File: packages/ocs/dashboards/block-pool/mirroring-card.scss [hunk 1]
```diff
@@ -25,3 +25,7 @@
     margin-right: var(--pf-global--spacer--sm);
   }
 }
+
+.odf-block-pool-utilization-card-body {
+  padding: 0 20px;
+}
```
Comment (bipuladh) [inline packages/ocs/dashboards/block-pool/mirroring-card.scss:31]: Are these changes enough for various common screen-sizes? QE might be testing it at another resolution.

Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/block-pool/mirroring-card.scss:31]: yes tested with other sizes, its working fine. The 20 pixels of padding on the right and left are also not required. The earlier "co-utilization-card__body" css class was not defined which causes this issue. I will maintain the same property "padding: 0"

**Review by bipuladh — APPROVED** [review]
/lgtm

**Comment by GowthamShanmugam** [comment]
/retitle Values on graph go out of tile

**Pull request merged**